routes/recaptcha_solver.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express')
+const router = express.Router()
+const utils = require('../helpers/utils')
+
+const DEFAULT_TIMEOUT = 1000;  // 1 second
+
+/*
+ * This module introduces new ability to puppeteer-service.
+ * It is capable of solving recaptchas on the given web-page.
+ * If there is no recaptcha on the page nothing bad will happen.
+ * If there is recaptcha it solves it and then inserts the special code
+ * into the page automatically.
+ *
+ * Returns useful information about recaptcha_solving.
+ * For more information about return value visit
+ * https://github.com/berstend/puppeteer-extra/tree/master/packages/puppeteer-extra-plugin-recaptcha#result-object
+ */
+
+/**
+ *
+ * @param page - page with possible recaptcha.
+ * @param request - request to the page.
+ */
+
+async function action(page, request) {
+    let recaptcha_data = await page.findRecaptchas();
+
+    if (request.body.solve_recaptcha) {
+        recaptcha_data = await page.solveRecaptchas();
```

Comment:
I'm not sure if this even possible. Sometimes we need to click "submit captcha" button, but in general case it is quite difficult to find it. So we most likely ask a user to click it yourself but it makes no sense in case because we want to "automatically" solve recaptcha. So I think there NO need and possibilities to create such a thing.